ndbench-api/src/main/java/com/netflix/ndbench/api/plugin/NdBenchAbstractClient.java

```diff
@@ -32,7 +32,7 @@
  *
  * @author vchella, pencal
  */
-public interface NdBenchAbstractClient<W> {
```

Comment:
while we are in here, lets discuss the naming of parameter W. W stands for write because it is the type returned by writeSingle but it is not descriptive of what the type is. Can we rename W to PostImageType or am I thinking about it too much?